CHANGELOG.rst

```diff
@@ -2,6 +2,31 @@
 Wijzigingen
 ===========
 
+1.3.0 (2022-07-7)
```

Comment:
is de finale set van wijzigingen? Kan ik nu het geheel wat Zaken API 1.3 gaat worden testen en bevindingen rapporteren?